site/docs/pages/identity/get-avatar.mdx

```diff
@@ -8,16 +8,44 @@ use it with Next.js or any Node.js backend.
 
 Supported providers:
 
+- [Basenames](https://www.base.org/names)
 - ENS
 
 ## Usage
 
-```tsx
+Get Basename avatar:
+
+:::code-group
+```tsx [code]
 import { getAvatar } from '@coinbase/onchainkit/identity';
+import { base, mainnet } from 'viem/chains';
+
+const baseAvatar = await getAvatar({ ensName: 'paulcramer.eth', chain: base });
+```
 
-const avatar = await getAvatar({ ensName: 'vitalik.eth' });
+```ts [return value]
+https://zku9gdedgba48lmr.public.blob.vercel-storage.com/basenames/avatar/paul.base.eth/1722120524815/FWUzoZmJ_400x400-kWjr2gMvjNe9hHMs9Z9LxGVGIME3By.jpg
```

Comment:
Interesting link

Reply:
I know haha. It's not very pretty but the user can copy and paste that link to go directly to the basename avatar